packages/components/src/button/style.scss

```diff
@@ -129,8 +129,11 @@
 			background: transparent;
 			transform: none;
 			opacity: 1;
-			box-shadow: none;
-			outline: none;
+
+			&:not(:focus) {
```

Comment:
What does this actually do? My mind warps trying to fully grok whether this is necessary or not.

Reply:
As far as I know, the design spec says that disabled secondary and tertiary buttons shouldn't have a border, which is the current situation. So there needs to be something to remove the `box-shadow` and `outline` defined higher up the cascade. 

But because of the specificity of the rule, this `:not(:focus)` block is needed to make sure that only happens when the button _doesn't_ have focus.